airflow/api_fastapi/core_api/serializers/tasks.py

```diff
@@ -0,0 +1,159 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import inspect
+from collections import abc
+from datetime import datetime, timedelta
+
+from pydantic import AliasGenerator, BaseModel, ConfigDict, computed_field, field_validator
+
+from airflow.models.mappedoperator import MappedOperator
+from airflow.serialization.serialized_objects import SerializedBaseOperator, encode_priority_weight_strategy
+from airflow.task.priority_strategy import PriorityWeightStrategy
+
+
+class TimeDeltaResponse(BaseModel):
+    """Time delta serializer for responses."""
+
+    object_type: str = "TimeDelta"
+    days: int
+    seconds: int
+    microseconds: int
+
+    model_config = ConfigDict(
+        alias_generator=AliasGenerator(
+            serialization_alias=lambda field_name: {
+                "object_type": "__type",
+            }.get(field_name, field_name),
+        )
+    )
```

Comment:
Why do we need this ?

Reply:
Currently, the `TimeDelta` schema has an `objectType` property which gets serialized as `__type`, code [here](https://github.com/apache/airflow/blob/main/airflow/api_connexion/schemas/common_schema.py#L41). So, we need this to maintain consistency with old API's `TimeDelta` schema, example in test [here](https://github.com/apache/airflow/blob/main/tests/api_connexion/endpoints/test_task_endpoint.py#L135).

Reply:
It'll be great if we can remove `__type` from the new API response, since I'm not sure it adds much value from user perspective (since it sort of becomes obvious that it's a timedelta object).